src/firefly/js/tables/TablesCntlr.js

```diff
@@ -600,6 +600,17 @@ function asyncFetch(request, hlRowIdx, dispatch, tbl_id) {
     });
 }
 
+/**
+ * @callback actionWatcherCallback
```

Comment:
can we put his line on all our watchers?  Maybe you already did it.  This is a great way to identify them.  I have been concerned about quickly recognizing a watcher.